Formula/k/kubehound.rb

```diff
@@ -0,0 +1,37 @@
+class Kubehound < Formula
+  desc "Tool for building Kubernetes attack paths"
+  homepage "https://kubehound.io"
+  url "https://github.com/DataDog/KubeHound/archive/refs/tags/v1.3.3.tar.gz"
+  sha256 "5ee3ca8621d4611e682ed2325bb12b87d6f00c4f7717195d5f43856f5d9b7917"
+  license "Apache-2.0"
+
+  depends_on "go" => [:build, :test]
+  depends_on "docker" => :test
+
+  def install
+    goos = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOOS").chomp
+    goarch = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOARCH").chomp
+
```

Comment:
Can't `go` determine this by itself?

Reply:
Yes, but the idea is to set the variables in the code at build time so it can display the information in the `version` command for instance (so when debugging it's easier know for which arch the binary was built for instance).
I asked the developers of the project to switch to `runtime.GOOS/GOARCH` in the code and this will be done in a future release.
Can this be kept like this for this release, and removed in the next one?